### ISSUE TITLE
hygieia 2.0.0 proposal

### ISSUE THREAD

**Issue by jdamick** [capitalone/Hygieia#197]
For hygieia 2.0 it has been proposed that we change hygieia from a snapshot in time to a real-time dashboard that incorporates dev pipeline graphs along with operational graphs.

Desired:
- [ ] near real-time dev pipeline dashboard metrics
- [ ] near real-time operational dashboard metrics
- [ ] customizable widgets with customizable queries
- [ ] alerting based on defined criteria
- [ ] pre-baked dashboards / templates (to match current hygieia functionality)

To get there I think we should take the following steps:

- [ ] change our DB to influxdb, which will be a better fit for our needs and give us integration flexibility
- [ ] change our UI components to be grafana
- [ ] build out dashboard templates for our canned dashboards
- [ ] let grafana handle altering (when it's done: https://github.com/grafana/grafana/issues/2209)
- [ ] update the collectors to send metrics into influxdb
- [ ] migrate existing data from mongo to influx

So basically hygieia becomes a project that houses various collectors and adds some opinionated templates to grafana/influxdb.

cc/ @tabladrum

**Comment by kapilt** [comment]
+1

**Comment by tabladrum** [comment]
Locked conversation and will unlock soon after gathering more information.

**Comment by tabladrum** [comment]
Closing this. Will stay the course an revisit this later.
